Autoescuela/Autoescuela.xcodeproj/project.pbxproj

```diff
@@ -287,6 +299,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = 8DURMPUBZ2;
 				INFOPLIST_FILE = Autoescuela/Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```

Comment:
Why are you targeting into ios 11? try to use the latest version.

Reply:
this is the minimum support IOS version